lnwallet/wallet.go

```diff
@@ -449,9 +457,12 @@ func (l *LightningWallet) handleFundingReserveRequest(req *InitFundingReserveMsg
 		return
 	}
 
+	capacity := req.LocalFundingAmt + req.RemoteFundingAmt
+	localFundingAmt := req.LocalFundingAmt
+
 	id := atomic.AddUint64(&l.nextFundingID, 1)
 	reservation, err := NewChannelReservation(
-		req.Capacity, req.FundingAmount, req.CommitFeePerKw, l, id,
+		capacity, localFundingAmt, req.CommitFeePerKw, l, id,
```

Comment:
Nit: could also make `NewChannelReservation` take the remote funding amount as well.

Reply:
Agreed, but seemed like a change too big to be done in this PR. Added a TODO.